crates/python/src/qpu/translation.rs

```diff
@@ -88,33 +90,59 @@ py_wrap_error!(
     PyRuntimeError
 );
 
+#[derive(Copy, Clone)]
+pub enum TranslationBackend {
+    V1,
+    V2,
+}
+
+py_wrap_simple_enum! {
+    PyTranslationBackend(TranslationBackend) as "TranslationBackend" {
+        V1,
+        V2
+    }
+}
+
 #[derive(Clone, Default)]
 #[pyclass(name = "TranslationOptions")]
-pub struct PyTranslationOptions(TranslationOptions);
+pub struct PyTranslationOptions {
+    inner: TranslationOptions,
+    backend: Option<TranslationBackend>,
+}
```

Comment:
Since `TranslationOptions` already includes the backend (...in fact, I think that's currently the _only_ option), why do we need to duplicate that?

Reply:
I'm also wondering the same thing